docs/guides/upgrading.rst

```diff
@@ -222,6 +232,12 @@ From the "users_entity" table, the ``password`` and ``hash`` columns have been r
 
 The ``geocode_cache`` table has been removed as it was no longer used.
 
+``subtype`` column in ``entities`` table no longer holds a subtype ID, but a subtype string
+``entity_subtypes`` table has been dropped.
+
+``subtype`` column in ``entities`` table no longer holds a subtype ID, but a subtype string
+``entity_subtypes`` table has been dropped.
```

Comment:
the same text twice? Or should a different table be referenced

Reply:
No, rebase madness